homeassistant/data_entry_flow.py

```diff
@@ -182,16 +184,16 @@ def async_has_matching_flow(
         )
 
     @callback
-    def async_get(self, flow_id: str) -> FlowResult:
+    def async_get(self, flow_id: str) -> _FlowResultT:
         """Return a flow in progress as a partial FlowResult."""
         if (flow := self._progress.get(flow_id)) is None:
             raise UnknownFlow
-        return _async_flow_handler_to_flow_result([flow], False)[0]
+        return _async_flow_handler_to_flow_result([flow], False)[0]  # type: ignore[return-value]
```

Comment:
I think `_async_flow_handler_to_flow_result` should be updated to use `_flow_result` and the generic type.

It also seems to belong as a method in the flow manager class and not a separate function.